### ISSUE TITLE
[Bug]: Addon doesn't start after initial setup.

### ISSUE THREAD

**Issue by benek984** [FaserF/hassio-addons#164]
### The problem

Was able to start add-on and access via http as https did not work. Specified email address and password and after this add-on doesn't start. Logs below.

### What version of Home Assistant Core has the issue?

2023.4.5

### What version of the addon has the issue?

latest

### What type of installation are you running?

Home Assistant OS

### Add-On causing the issue

Wiki.JS

### Log information

```txt
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
You have activated SSL. SSL Settings will be applied
This add-on requires the MariaDB core add-on 2.0 or newer!
Starting Wiki.JS
Loading configuration from /wiki/config.yml... OK
2023-05-02T20:16:58.529Z [MASTER] info: =======================================
2023-05-02T20:16:58.531Z [MASTER] info: = Wiki.js 2.5.298 =====================
2023-05-02T20:16:58.532Z [MASTER] info: =======================================
2023-05-02T20:16:58.532Z [MASTER] info: Initializing...
2023-05-02T20:16:59.580Z [MASTER] info: Using database driver mysql2 for mariadb [ OK ]
2023-05-02T20:16:59.587Z [MASTER] info: Connecting to database...
2023-05-02T20:16:59.643Z [MASTER] info: Database Connection Successful [ OK ]
2023-05-02T20:17:00.624Z [MASTER] warn: Mail is not setup! Please set the configuration in the administration area!
2023-05-02T20:17:00.803Z [MASTER] info: Loading GraphQL Schema...
2023-05-02T20:17:01.763Z [MASTER] error: Package subpath './public/extractFiles' is not defined by "exports" in /wiki/node_modules/extract-files/package.json
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
```

**Comment by FaserF** [comment]
Please give it another try with the latest release. But most likely this is a dependency issue within wiki.js itself

**Comment by veilofsecurity** [comment]
Same issue here with latest release. See some stuff around about incorrect NodeJS versions but that shouldn't apply to Docker version. Also see some stuff about ssl causing it but tried without and same error.

**Comment by benek984** [comment]
Hi, still the same:

````
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
This add-on requires the MariaDB core add-on 2.0 or newer!
Starting Wiki.JS
Loading configuration from /wiki/config.yml... OK
2023-05-29T10:12:19.834Z [MASTER] info: =======================================
2023-05-29T10:12:19.837Z [MASTER] info: = Wiki.js 2.5.298 =====================
2023-05-29T10:12:19.837Z [MASTER] info: =======================================
2023-05-29T10:12:19.838Z [MASTER] info: Initializing...
2023-05-29T10:12:21.373Z [MASTER] info: Using database driver mysql2 for mariadb [ OK ]
2023-05-29T10:12:21.378Z [MASTER] info: Connecting to database...
2023-05-29T10:12:21.414Z [MASTER] info: Database Connection Successful [ OK ]
2023-05-29T10:12:22.570Z [MASTER] warn: Mail is not setup! Please set the configuration in the administration area!
2023-05-29T10:12:22.741Z [MASTER] info: Loading GraphQL Schema...
2023-05-29T10:12:23.823Z [MASTER] error: Package subpath './public/extractFiles' is not defined by "exports" in /wiki/node_modules/extract-files/package.json
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
````

**Comment by FaserF** [comment]
Wiki.JS got an update, please try it again with version 1.5.1 of the addon

**Comment by benek984** [comment]
Hi I think now getting new error and addon stops:

````
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
You have activated SSL. SSL Settings will be applied
This add-on requires the MariaDB core add-on 2.0 or newer!
Starting Wiki.JS
You're using an unsupported Node.js version. Please read the requirements.
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped

````

**Comment by veilofsecurity** [comment]
It says pretty explicitly that WikiJS does not work with Node.js 18+ and that there is no plans for WikiJS 2.x to support. WikiJS 3+ will support it but it's early stages still. The add-on image needs reverted back to Node.js 16.x to work.

**Comment by FaserF** [comment]
The issue is fixed with version 1.5.2
Unfortunatly a version bump of the OS within the addon did increase the NodeJS Version to an unsupported version, since NodeJS 16 has been dropped to be supported by the OS.
I have now manually implemented the older NodeJS Version and now it works again.

**Comment by veilofsecurity** [comment]
Looks good now, thanks!
